python/openassetio/test/managerValidator/_implementation.py

```diff
@@ -0,0 +1,151 @@
+#
+#   Copyright 2013-2021 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Boilerplate classes for initialization and execution of test cases.
+"""
+import unittest
+
+from openassetio import hostAPI, pluginSystem, logging
+
+
+__all__ = ['createHarness']
+
+
+def createHarness(managerIdentifier):
+    """
+    Create the test harness used begin test case execution.
+    """
+    hostInterface = _ValidatorHarnessHostInterface()
+    logger = logging.SeverityFilter(logging.ConsoleLogger())
+    managerFactory = pluginSystem.PluginSystemManagerFactory(logger)
+
+    session = hostAPI.Session(hostInterface, logger, managerFactory)
+    session.useManager(managerIdentifier)
+
+    loader = _ValidatorTestLoader(session)
+    return _ValidatorHarness(unittest.main, loader)
+
+
+class _ValidatorHarness:
+    """
+    Facade wrapping the unittest runner.
```

Comment:
[consider][minor] My bad, it's not really a "facade", or at least not only facade, more like an "adapter", or maybe we should be hand-wavy and just say "wrapper".

Reply:
Hows [this](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/1761c2208e3871c94b14e5d11c1634c15083280d)?